content/posts/weekly/May-2023.md

```diff
@@ -23,3 +23,11 @@ any significant new features.
 - We have implemented a denylisting mechanism allowing us to denylist namespaces/projects to prevent misuse of our service. ([packit-service#2046](https://github.com/packit/packit-service/pull/2046))
 - Packit will now additionally require for each test job requiring build a build job definition to be present in the Packit configuration file. ([packit-service#2012](https://github.com/packit/packit-service/pull/2012))
 - Packit now checks whether the configured architecture for the test target is supported by Testing Farm and in case it's not supported, Packit doesn't submit these test requests. ([packit-service#2053](https://github.com/packit/packit-service/pull/2053))
+
+## Week 21 (May 23rd – May 29th)
+
+- Unsuccessful Image Builder requests now provide error details so you can fix the Image configuration. ([packit#1981](https://github.com/packit/packit/pull/1981))
+- Copr projects created by Packit will not follow the Fedora branching from now. This decision has been made to lower the load on Copr from the temporary Copr projects created, mainly, for the PR builds. If you are releasing your packages to the Copr, please use the new setting `follow_fedora_branching`.
+  Already existing projects are not affected by this change and it is also not enforced with the custom Copr repositories. ([packit#1970](https://github.com/packit/packit/pull/1970))
+- Now `additional_repos` dependencies will not be broken by the `tf_extra_params` option ([packit-service#2056](https://github.com/packit/packit-service/pull/2056))
```

Comment:
Additional artifact(s) passed as `artifacts` in the `tf_extra_params` dictionary will be now combined with the artifact passed by Packit instead of rewriting it.